### DISCUSSION TITLE
Upload an image into a rowy image field without using rowy

### DISCUSSION THREAD

**Discussion by javobal** [AntlerVC/firetable#1274] [Support / Q&A] [answered]
I would like to upload profile pictures from my mobile app into an Image field, i was thinking of directly upload it to firestorage and then modify the value directly in firestore just like rowy fills it. Is there an easier or maybe already available way ? Thanks

**Comment by BeeBombshell** [comment, marked as answer]
Hey @javobal. Yes, you can indeed upload profile pictures from your mobile app to Firebase Storage and store the download URLs of the images in Firestore. This approach is commonly used and can be implemented easily. Here's our documentation demonstrating how you can achieve this: https://docs.rowy.io/integrations/flutterflow/displaying-images.
